committers-credo.md

```diff
@@ -4,6 +4,8 @@ Some people have write access to WP-CLI repositories. These people are identifie
 
 * [maintainers](https://github.com/orgs/wp-cli/teams/maintainers) - Project leadership
 * [committers](https://github.com/orgs/wp-cli/teams/committers) - Trusted contributors
+ 
+    The above links are not public; they only work for GitHub users who are part of the WP-CLI GitHub organization.
```

Comment:
@schlessera @danielbachhuber @swissspidy  I'm uncertain about whether these teams should be visible to the public. If they should be, we'll need to adjust their visibility settings, as outlined here: https://docs.github.com/en/organizations/organizing-members-into-teams/changing-team-visibility.

Reply:
They are already _visible_. The problem is even _visible_ teams are only visible to organization members, not the general public. Teams cannot be made _publicly visible_. That's also what that documentation says.

Given that this page is for committers and committers are part of the GitHub org anyway, I don't really see an issue with that though.

Reply:
Committers and maintainers links are used in several other pages also, like governance, code review, etc. It would be nice thing to be able to view who are maintaining the project by non-team members also.

Reply:
"People" shows the nine members without being logged in to GitHub, thirteen when I'm logged in.

https://github.com/orgs/wp-cli/people